### ISSUE TITLE
Problems in JP web page - Ignite UI for Angular - Demo

### ISSUE THREAD

**Issue by pnishimura** [IgniteUI/igniteui-docfx#271]
Yoshimi informed a following page to JP members today,
Then I found 2 problems.

https://jp.infragistics.com/products/ignite-ui-angular/angular/components/grid.html

1. Side menu is not displayed.
2. Click and open STACKBLITZ, but it shows blank page due to XSS error.

Please find the attached file to get more detail.

Thanks,
Paul

![problem](https://user-images.githubusercontent.com/29943424/38712561-7d2a8372-3f07-11e8-9326-dacaa3cbcf9d.PNG)

**Comment by ig-ealbert** [comment]
@pnishimura I'm assuming this is browser specific to IE?  Side menu works in Chrome.  I think to view the StackBlitz samples I had to unblock it in my browser.  @kdinev can investigate in IE and see if this is expected.

**Comment by pnishimura** [comment]
@ig-ealbert 
Thanks for your reply! Yes, those 2 problems occurred in only IE11, not Chrome.

IE11 is one of major desktop browsers here in Japan,
So we Tokyo team believe that we'd better support IE11 as well as other browsers.

http://gs.statcounter.com/browser-version-market-share/desktop/japan/#monthly-201703-201803-bar
![browser](https://user-images.githubusercontent.com/29943424/38787977-2e226e36-416c-11e8-8c6c-2016993af9fb.PNG)

**Comment by ig-ealbert** [comment]
@tiliev @rkaraivanov @zdrawku @pnishimura I just tested this with the live site in IE11.  The left menu is fixed.  However, I still cannot open StackBlitz links because of XSS.  Is that intentional?  I know there is an option in IE11 to disable that, but is that what we are advising customers to do?  I didn't see any discussion here about XSS, which is why I'm asking.

**Comment by pnishimura** [comment]
Hi @ig-ealbert 
Thanks for your testing!

I just also tested this in JP production site in IE11.
https://jp.infragistics.com/products/ignite-ui-angular/angular/components/grid.html

As same as you reported above,
The left menu is fixed. However, I still cannot open StackBlitz links because of XSS.

Hope this will be fixed as soon as possible.

**Comment by tiliev** [comment]
I am reopening it, will investigate it and will let you guys know.

**Comment by tiliev** [comment]
The issue is fixed and released, but StackBlitz has some issues and does not display any samples currently. I logged the bug in their repo - https://github.com/stackblitz/core/issues/461.

**Comment by pnishimura** [comment]
@tiliev Thanks for your prompt action!

**Comment by ig-ealbert** [comment]
The issue logged on StackBlitz was fixed, and the page loads now.  Could you please confirm?  I see the files list and the code editor, but not the live sample.  Might just be that it takes many minutes to load.

**Comment by ig-ealbert** [comment]
Error occurred:

Unable to get property 'match' of undefined or null reference
 Instantiating https://unpkg.com/@angular/animations@5.2.10/bundles/animations-browser.umd.js
 Loading https://unpkg.com/@angular/animations@5.2.10/browser.js
 Loading https://unpkg.com/@angular/platform-browser@5.2.10/bundles/platform-browser-animations.umd.js
 Loading https://unpkg.com/@angular/platform-browser@5.2.10/animations.js
 Loading https://stackblitz.com/catch/app/app.module.ts
 Loading https://stackblitz.com/catch/main.ts
 Loading blitzapp

**Comment by zdrawku** [comment]
Tihomir is currently investigating it, thank you Liz.

**Comment by tiliev** [comment]
@pnishimura, @ig-ealbert, 
We eliminated all of the reported issues here. Feel free to test them in staging. 
What's left is the issue with the grid's width as reported [here](https://github.com/IgniteUI/igniteui-angular/issues/1330). Is it OK to close the current issue?

**Comment by ig-ealbert** [comment]
@tiliev Grid loads on StackBlitz now, thank you!  I'd wait for @pnishimura to confirm, and then the issue can be closed.

**Comment by ig-ealbert** [comment]
@pnishimura Tihomir indicated that this is logged separately: https://github.com/IgniteUI/igniteui-angular/issues/1330

**Comment by zdrawku** [comment]
@pnishimura if you consider this issue as closed, you may close it. As @ig-ealbert mentioned, another issue is logged to the grid width in iframe.

**Comment by pnishimura** [comment]
Hi guys,
Now I totally understood our situation and confirmed this #271 has been resolved.
So I closed this now.
Thanks you so much, guys!
